Mathlib/Topology/SubsetProperties.lean

```diff
@@ -29,10 +30,11 @@ We define the following properties for sets in a topological space:
 * `IsClopen`: a set that is both open and closed.
 * `IsIrreducible`: a nonempty set that has contains no non-trivial pair of disjoint opens.
   See also the section below in the module doc.
+* `IsSigmaCompact`: a set that is the union of countably many compact sets.
 
 For each of these definitions (except for `IsClopen`), we also have a class stating that the whole
 space satisfies that property:
-`CompactSpace`, `PreirreducibleSpace`, `IrreducibleSpace`.
+`CompactSpace`, `PreirreducibleSpace`, `IrreducibleSpace`. (See below for `SigmaCompactSpace`.)
```

Comment:
```suggestion
`CompactSpace`, `PreirreducibleSpace`, `IrreducibleSpace`, `SigmaCompactSpace`.
```
and remove the mention below. I think it keeps consistency with the rest of the docstring.